### ISSUE TITLE
Computed values provide incorrect link

### ISSUE THREAD

**Issue by niladam** [causelabs/resource-index-link#4]
Hello,

I'm using this like so:

```php

ResourceIndexLink::make('Full Name', function () {
    return "{$this->first_name} {$this->last_name}";
})
    ->newTab()
    ->onlyOnIndex()
    ->sortable(),

```

The generated link is just the nova path -- without any resource whatsoever. Am i doing something wrong ?

**Comment by rick-hidef** [comment]
It looks like Nova doesn't pass the resource ID to the field for computed fields. A work-around would be to add a mutator to your base model, and use that mutator instead of a custom Nova field. For example:
```
// app/User.php - Base Model
public function getFullNameAttribute()
{
    return $this->first_name . ' ' . $this->last_name;
}
```

```
// app/Nova/User.php - Nova Resource Model
ResourceIndexLink::make('Full Name')
    ->sortable()
    ->rules('required', 'max:255')
    ->newTab()
    ->onlyOnIndex(),
```

I'll look to see if there's a way to grab that ID from Nova, but this should get you going in the mean time.

**Comment by niladam** [comment]
Using a mutator is clearly the most efficient solution -- however, i think that this shouldn't impose on using one and/or provide some info about this -- and suggest using a mutator.

Thank you for your package!:)

**Comment by rick-hidef** [comment]
@niladam I made a modification that should allow Computed Fields to work without the workaround mentioned above. Released in v1.0.3. Let me know if you run into any issues.
